example/lib/star_wars/models.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:built_collection/built_collection.dart';
+import 'package:built_value/built_value.dart';
+import 'package:built_value/serializer.dart';
+import 'package:built_value/standard_json_plugin.dart';
+
+part 'models.g.dart';
+
+/// json serializer to build models
+@SerializersFor([
+  PlanetPageModel,
+  PlanetModel,
+])
+final Serializers serializers =
+    (_$serializers.toBuilder()..addPlugin(StandardJsonPlugin())).build();
+
+/// equals one page
+abstract class PlanetPageModel
+    implements Built<PlanetPageModel, PlanetPageModelBuilder> {
+  PlanetPageModel._();
+
+  /// default factory
+  factory PlanetPageModel([void Function(PlanetPageModelBuilder) updates]) =
+      _$PlanetPageModel;
+
+  /// serialize the model
+  static Serializer<PlanetPageModel> get serializer =>
+      _$planetPageModelSerializer;
+
+  /// url to next page
```

Comment:
Are all these comments because there's a lint rule for public members to be documented?
If so, we can consider removing them and disabling this lint for the example folder

These comments make me feel like "thanks captain obvious" 😛

Reply:
Maybe copy-paste the root analysis-options.yaml on the example folder, or add a `// ignore-for-file` on the top of the file

Reply:
Haha, yes they are "captain obvious" ... I think i will add a linter rule on top of the file :)